base/invalidation.jl

```diff
@@ -160,6 +162,8 @@ function invalidate_code_for_globalref!(b::Core.Binding, invalidated_bpart::Core
                 end
             end
         end
+        # This assumes that we haven't gotten here unless something needed to be invalidated
+        ccall(:jl_maybe_log_binding_invalidation, Cvoid, (Any,), invalidated_bpart)
```

Comment:
This seems potentially problematic, because the invalidated code instances will get interleaved with nested invalidations from other bindings. Can SnoopCompile disambiguate this?

Reply:
Also should probably check if anything was actually invalidated by adding appropriate boolean returns.

Reply:
Thanks, that's exactly the answer I was fishing for.